packages/patternfly-4/react-core/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/Card/card.css';
+import { css } from '@patternfly/react-styles';
+
+export interface CardProps extends React.HTMLProps<HTMLDivElement> {
+  /** content rendered inside the Card */
+  children?: React.ReactNode; 
+  /** additional classes added to the Card */
+  className?: string; 
+  /** Sets the base component to render. defaults to article */
+  component?: React.ReactNode;
```

Comment:
This was componentShape before so it will break JS.

Reply:
JS gets `Proptypes.Node` in our generated dist. PropTypes says:
>  // Anything that can be rendered: numbers, strings, elements or an array
>  // (or fragment) containing these types.
>  // see https://reactjs.org/docs/rendering-elements.html for more info
>  optionalNode: PropTypes.node,

in [their docs](https://github.com/facebook/prop-types), so I think this is the correct type. I'm curious to see an example that is valid and works but PropTypes gets upset at!